scripts/ci/determine-tests.mjs

```diff
@@ -202,11 +215,10 @@ try {
     setOutput(outputNameTestsToRun, '');
     setOutput(outputNameShardIndex, [0]);
     setOutput(outputNameShardTotal, [0]);
-  }
-
-  if (
+  } else if (
     error instanceof DependentPackageChangeError ||
-    error instanceof PackageJsonChangeError
+    error instanceof PackageJsonChangeError ||
+    error instanceof RunAllTestsInMergeQueueError
```

Comment:
Let's simplify :)
Add an abstract class in the proto chain that all those errors extends, instead of them extending.
Then, do an instanceof on this abstract class instead of having 3 instanceof.

Reply:
Personally, I like it better like it is now. It is way more explicit. Why add more abstraction and complexity in a simple script ?

Reply:
The less lines the better 🤷